### PR TITLE
v17.0.1

### PR DESCRIPTION
**Pull request by Faraz32123**

### CODE REVIEW

File: tutorcredentials/__about__.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "17.0.0"
+__version__ = "17.0.1"
```
Comment (regisb) [inline tutorcredentials/__about__.py:1]: You, my friend, forgot to collect changelog entries ;) https://github.com/overhangio/tutor/pull/991/files#diff-f94b76ecc373d484f1bf8b218e690f92be03225e3a5940f6f0d392b816783901

Comment (Faraz32123) [reply tutorcredentials/__about__.py:1]: Actually, there were no changelog entries created initially, No changelog fragments to collect.

Comment (regisb) [reply tutorcredentials/__about__.py:1]: Good point! There should have been an entry in this PR though: https://github.com/overhangio/tutor-credentials/pull/31

Comment (Faraz32123) [reply tutorcredentials/__about__.py:1]: forgot to comment on latest PR that is merged for the changelog entries.

Comment (Faraz32123) [reply tutorcredentials/__about__.py:1]: yup!

Comment (regisb) [reply tutorcredentials/__about__.py:1]: It's OK. I just pushed a changelog entry to master.

Comment (Faraz32123) [reply tutorcredentials/__about__.py:1]: thanks regis.

**Pull request merged**